docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from 'react';
+import SwipeableViews from 'react-swipeable-views';
+import Box, { BoxProps } from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import SvgProductCore from 'docs/src/icons/SvgProductCore';
+import SvgProductAdvanced from 'docs/src/icons/SvgProductAdvanced';
+import SvgProductTemplates from 'docs/src/icons/SvgProductTemplates';
+import SvgProductDesign from 'docs/src/icons/SvgProductDesign';
+import SvgMuiX from 'docs/src/icons/SvgMuiX';
+
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+
+function ProductItem({
+  icon,
+  name,
+  description,
+}: {
+  icon: React.ReactNode;
+  name: React.ReactNode;
+  description: React.ReactNode;
+}) {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: { md: 'center' },
+        p: 2,
+        flexDirection: { xs: 'column', md: 'row' },
+      }}
+    >
+      <Box sx={{ mr: 2 }}>{icon}</Box>
+      <Box>
+        <Typography color="text.primary" variant="body2" fontWeight="bold">
+          {name}
+        </Typography>
+        <Typography color="text.secondary" variant="body2" fontWeight="regular">
+          {description}
+        </Typography>
+        <Typography
+          color="primary"
+          variant="body2"
+          fontWeight="bold"
+          sx={{ display: 'flex', alignItems: 'center', minHeight: 24 }}
+        >
+          <span>Learn more</span>{' '}
+          <KeyboardArrowRightRounded fontSize="small" sx={{ mt: '2px', ml: '2px' }} />
+        </Typography>
+      </Box>
+    </Box>
+  );
+}
+
+function Highlight({
+  children,
+  selected = false,
+  onClick,
+  sx,
+}: {
+  children: React.ReactNode;
+  selected?: boolean;
+  onClick?: BoxProps['onClick'];
+  sx?: BoxProps['sx'];
+}) {
+  return (
+    <Box
+      role="button"
+      onClick={onClick}
+      sx={{
+        borderRadius: 1,
+        transition: '0.3s',
+        height: '100%',
```

Comment:
I'm wondering if we should set the height 100%. On my prototype, I haven't. The effect of not doing that is that the ProductsSwitcher doesn't stay bigger than the other column. Maybe we could iterate here.

Reply:
The height 100% is for mobile viewport only.